src/plugins/condition/ConditionManager.js

```diff
@@ -220,7 +230,7 @@ export default class ConditionManager extends EventEmitter {
             this.conditionSetDomainObject.configuration.conditionCollection.unshift(newCondition);
         }
 
-        this.initCondition(newCondition, index);
+        this.initCondition(newCondition, index, true);
```

Comment:
may be can call 'init' and then 'update'?

Reply:
hmm, ll it trigger mutate?

Reply:
no it won't